Docs/CLI.md

```diff
@@ -2,22 +2,46 @@
 
 YAFC can be invoked via command line:
 
-`yafc PROJECT_FILE DATA_DIR MODS_DIR [expensive]`
+`YAFC [<data-path>] [--mods-path <path>] [--project-file <path>] [--expensive] [--help]`
```

Comment:
```suggestion
`YAFC [<data-path> [--mods-path <path>] [--project-file <path>] [--expensive]] [--help]`
```

Reply:
Yeah, that makes it cleared that there are two modes of operation.